torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2543,6 +2543,22 @@ at::Tensor& XLANativeFunctions::random_(
   return self;
 }
 
+at::Tensor& XLANativeFunctions::randperm_out(int64_t n, c10::optional<at::Generator> generator, at::Tensor & out) {
+  XLA_FN_COUNTER("xla::");
+  if (generator.has_value() && generator->defined()) {
```

Comment:
did you verify that this code block works as expected in your local tests?

Reply:
Yes, added a unit test for it.